### ISSUE TITLE
Plots made with Pandas AI not appearing in Streamlit

### ISSUE THREAD

**Issue by programmingninjas** [programmingninjas/GPT-3.5-ON-STEROIDS#17]
In the feature of file analysis. We are using PandasAI for the analysis of Excel files. So plots are not appearing in the streamlit frontend even after using the StreamlitMiddleware of pandasai.

**Comment by Rani1303** [comment]
Hey @programmingninjas I'm interested in addressing this issue and would like to contribute to the resolution.

**Comment by programmingninjas** [comment]
@Rani1303  sure you can! Kindly set up the project locally and make sure you understand it thoroughly. I will assign this issue to you after 1st Jan as SWOC contributions will start from then. Good Luck!

**Comment by Rani1303** [comment]
Thanks

**Comment by programmingninjas** [comment]
@Rani1303 Assigned you the issue! Start working on them

**Comment by k-Rohit** [comment]
Hey @programmingninjas I would like to address this issue and solve the bug

**Comment by programmingninjas** [comment]
@k-Rohit  This issue is already assigned to @Rani1303

**Comment by k-Rohit** [comment]
Is there any thing else I can contribute to?

**Comment by programmingninjas** [comment]
@k-Rohit yeah you can create a issue by yourself adding a new feature which you might think will be useful for the project.

**Comment by programmingninjas** [comment]
@Rani1303 updates?

**Comment by Cosmyccc** [comment]
Hey! I am unable to test my code due to Open API key limit exceed.

**Comment by programmingninjas** [comment]
@Rani1303 you have to create a new openai account with a new number in order to get a new key which will be free for 3 months

**Comment by Aryanmartinian** [comment]
Hi I would like to Contribute in this project under SWOC .

**Comment by programmingninjas** [comment]
@Aryanmartinian Sure Aryan Go for it
